File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False
        sa = [c for c in s]
        ta = [c for c in t]
        sa.sort()
        ta.sort()

        for i in range(len(s)):
            if sa[i] != ta[i]:
                return False
            
        return True

File: 242-valid-anagram/valid-anagram.go
func isAnagram(s string, t string) bool {
    if len(s) != len(t) {
        return false
    }
	var sa = []rune(s)
	var ta = []rune(t)

	sort.Slice(sa, func(i, j int) bool { return sa[i] > sa[j] })
	sort.Slice(ta, func(i, j int) bool { return ta[i] > ta[j] })
	for i, chr := range sa {
		if chr != ta[i] {
			return false
		}
	}
	return true

}